package.json

```diff
@@ -129,21 +143,22 @@
     "build:lib": "rollup -c rollup.config.lib.js",
     "build:dist:dev": "cross-env NODE_ENV=development rollup -c rollup.config.dist.js",
     "build:dist:prod": "cross-env NODE_ENV=production rollup -c rollup.config.dist.js",
-    "build:dist": "npm run build:dist:dev && npm run build:dist:prod",
-    "build": "npm run build:data && npm run build:lib && npm run build:dist",
+    "build:dist": "yarn run build:dist:dev && yarn run build:dist:prod",
+    "build": "yarn run build:data && yarn run build:lib && yarn run build:dist",
     "react:clean": "rimraf node_modules/{react,react-dom,react-addons-test-utils}",
-    "react:14": "npm run react:clean && npm i react@^0.14 react-dom@^0.14 react-addons-test-utils@^0.14",
-    "react:15": "npm run react:clean && npm i react@^15 react-dom@^15",
+    "react:14": "yarn run react:clean && yarn add -D react@^0.14 react-dom@^0.14 react-addons-test-utils@^0.14 --ignore-scripts",
+    "react:15": "yarn run react:clean && yarn add -D react@^15 react-dom@^15 react-test-renderer@^15 --ignore-scripts",
+    "react:16": "yarn run react:clean && yarn add -D react@^16.3.1 react-dom@^16.3.3 react-test-renderer@^16.3.1 --ignore-scripts",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "test": "jest --coverage --verbose",
-    "test:react": "npm run react:14 && jest && npm run react:15 && jest",
-    "test:all": "npm run lint && npm run test && npm run test:react",
+    "test:react": "yarn run react:14 && jest && yarn run react:15 && jest && yarn run react:16 && jest",
+    "test:all": "yarn run lint && yarn run test && yarn run test:react",
     "test:watch": "jest --watch",
     "test:perf": "cross-env NODE_ENV=production babel-node test/perf",
-    "examples:install": "babel-node scripts/examples npm install",
-    "examples:link": "npm link && babel-node scripts/examples npm link react-intl",
-    "preversion": "npm run clean && npm run build && npm run test:all",
-    "prepare": "npm run clean && npm run build"
+    "examples:install": "babel-node scripts/examples yarn install",
+    "examples:link": "yarn link && babel-node scripts/examples yarn link react-intl",
+    "preversion": "yarn run clean && yarn run build && yarn run test:all",
+    "prepare": "yarn run clean && yarn run build"
```

Comment:
why yarn, not npm ?

Reply:
to consolidate?  the main repo have `yarn.lock` file... actually, I'm also not sure why `yarn` if it was used `npm`, here don't have a workspace.

Reply:
Yep, the repo was on yarn before or switched to yarn while I was working on it - not even sure - so I had to change it.

I'm usually using npm myself, but whatever. For library consumers it makes no difference after all as far as I know.

Reply:
yarn was before my time. I would prefer to just use npm though.

Reply:
So we change it back to npm?

Reply:
Unless this is causing issues with this PR, I would prefer that be done in a separate one. This PR is already 76 commits and 42 files changed 👀 

<img width="150" alt="Screen Shot 2019-04-29 at 11 26 23 AM" src="https://user-images.githubusercontent.com/193272/56917949-a397e400-6a71-11e9-8b4b-6c134590cd68.png">